### DISCUSSION TITLE
error while processing response

### DISCUSSION THREAD

**Discussion by RedWilliams** [winder/Universal-G-Code-Sender#1768] [Help]
OK I'm pretty new to this world so please bare with me. I just purchased a Sainsmart 3018 prover. Then installed some upgrades.  The 3040, then added 500W 52mm air cooled spindle with it own power supply.  Along with Carveco maker I'm using UGS rather than Candle. 

So, get the project ready to go.  Everything looks good, I push the go button and turn on the spindle motor.  Within 90 seconds I get a bunch of the above errors and the CNC stops running.

After lots of head scratching I re zero the machine but I raise the zero of the Z axis and push the go button but I do not turn on the spindle that is powered by its own power supply.  No problems and its been running a for 30 minutes.

There is definitely a problem with either the motor, power supply, or the pot.  How do I figure which is the problem and what do I do about it?

**Comment by breiler** [comment]
Sorry about the late reply, I thought that I answered this... 🤔 

This is usually caused by electric interference (EMI) and you should make sure your controller i properly shielded and have a proper USB cable (preferably with those iron ferrite clumps on it).
